src/scripts/main.js

```diff
@@ -1,3 +1,44 @@
 'use strict';
 
-// write code here
+const parents = document.querySelector('ul');
+const listArr = [...document.querySelectorAll('li')];
+
+function sortList(list) {
+  const sortedList = list.sort((a, b) => {
+    return (
+      convertToNumber(b.dataset.salary) - convertToNumber(a.dataset.salary)
+    );
+  });
+
+  parents.innerHTML = '';
+
+  sortedList.forEach((item) => {
+    parents.appendChild(item);
```

Comment:
Directly manipulating the DOM in a loop can lead to performance issues, especially with large lists. Consider using a document fragment to batch DOM updates for better performance.